pkg/api/login_oauth.go

```diff
@@ -190,6 +190,23 @@ func (hs *HTTPServer) OAuthLogin(ctx *m.ReqContext) {
 		}
 	}
 
+	orgToRoleMap := connect.OrgToRoleMap()
+
+	for _, group := range userInfo.Groups {
```

Comment:
Make sure `userInfo.Groups` isn't `nil` for all non-generic-oauth social providers.

Reply:
Updated in the providers.

Reply:
closing since this value is marshalled from json

Reply:
Since the userInfo struct is unmarshalled from the JSON of the `/userinfo` endpoint, the Groups attribute, if not included is an empty slice.